Tests/configure_and_test_integration_instances.py

```diff
@@ -983,9 +983,12 @@ def get_pack_ids_to_install():
         #  END CHANGE ON LOCAL RUN  #
 
 
-def nightly_install_packs(build, install_method=install_all_content_packs, pack_path=None, service_account=None):
+def nightly_install_packs(build, install_method=None, pack_path=None, service_account=None):
```

Comment:
why did you remove the default?
this basically means you can change it to a positional argument now

Reply:
Yes, this default value is out of use during all calls to this function